subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/DependencyConstraintHandlerScope.kt

```diff
@@ -70,7 +96,7 @@ private constructor(
      * @return The dependency constraint.
      * @see [DependencyConstraintHandler.add]
      */
-    operator fun Configuration.invoke(dependencyConstraintNotation: Any): DependencyConstraint? =
+    operator fun Configuration.invoke(dependencyConstraintNotation: Any): DependencyConstraint =
```

Comment:
:x: This is an unnecessary change, please revert

Reply:
Reverted and fixed DCO.